template/src/ontology/Makefile.jinja2

```diff
@@ -1519,6 +1519,25 @@ validate-all-tsv: $(ALL_TSV_FILES)
 # Editors Utilities
 # ----------------------------------------
 
+# The merge template workflow is commonly used when ontology content is curated in a ROBOT template
+# with the intention to merge it directly into the editors file. The idea is to either create a template
+# manually or create a rule in {{ project.id }}.Makefile to download a template from, e.g., a Google Sheet, and then run
+# `sh run.sh make MERGE_TEMPLATE=tmp/merge_my_template.tsv merge_template` to merge the template into the editors file or
+# `sh run.sh make TEMPLATE_URL=http://example.org/merge_my_template.tsv merge_template` to download and then merge the template
+
+MERGE_TEMPLATE=tmp/merge_template.tsv
+MERGE_TEMPLATE_OPTIONS= --merge-before
+TEMPLATE_URL=
+
+# The tmp/merge_template.tsv is hardcoded as placeholder for downloading template files
+# from a remote location
+tmp/merge_template.tsv:
+	wget "$(TEMPLATE_URL)" -O $@
```

Comment:
I’d consider actively checking if `TEMPLATE_URL` is defined, and error out with an explicit error message in that case (something like “No URL defined for merging remote template, please define TEMPLATE_URL” or similar), rather than letting the rule fail with the error message from `wget` (when given an “empty” URL, `wget` spits a `http://: invalid hostname` message, which is correct from its point of view but not very helpful for an ODK user).

Reply:
Yes, of course: https://github.com/INCATools/ontology-development-kit/commit/afdb87143fb378d82d230b268459a2422e604264#diff-ab539569446338e97f29dddc47eccfb63d931e4efa0157d240796a7e9add1d68